demos/dashboards-find-similar-screener/demo.js

```diff
@@ -0,0 +1,129 @@
+const loadingLabel = document.getElementById('loading-label');
+
+function displayFindSimilarScreener (postmanJSON) {
+    const secIds = [
+        'secId',
+        'name',
+        'riskRating',
+        'ongoingCharge'
+    ];
+
+    const headerFormats = {
+        secId: 'Security ID',
+        name: 'Investment Name',
+        riskRating: 'Risk Rating',
+        ongoingCharge: 'Ongoing Charge'
+    };
+
+    const columns = secIds.map(id => ({
+        id: `InvestmentScreener_${id}`,
+        header: {
+            format: headerFormats[id] || id
+        }
+    }));
+
+    const board = Dashboards.board('container', {
+        dataPool: {
+            connectors: [
+                {
+                    id: 'find-similar-screener',
+                    type: 'MorningstarInvestmentScreener',
+                    options: {
+                        page: 1,
+                        pageSize: 20,
+                        languageId: 'en-GB',
+                        currencyId: 'USD',
+                        securityDataPoints: secIds,
+                        universeIds: ['FOESP$$ALL'],
+                        sortOrder: 'Name+Asc',
+                        filters: [
+                            {
+                                dataPointId: 'CategoryId',
+                                comparatorCode: 'EQ',
+                                // The returned list will be similar to the
+                                // chosen fund
+                                value: 'EUCA000591'
```

Comment:
Good. Please add also the title to the Grid, so it's clear to which fund we are comparing.